docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -1,13 +1,9 @@
 # Showcase
 
-<p class="description">The following is a list of some of the public apps using MUI.</p>
+<p class="description">Check out these public apps using Material UI to get inspired for your next project.</p>
 
-Are you using MUI? [Share your product](https://github.com/mui/material-ui/issues/22426).
+Are you working with MUI's component libraries? [Show us what you're building](https://github.com/mui/material-ui/issues/22426)! We'd love to see it.
 
-The websites we include rank well in at least one of these dimensions:
-
-- Non Material Design customization
```

Comment:
I have moved this information to https://www.notion.so/mui-org/Showcase-8ab60cbd5afc45729c60d9fcf00e8838#a887c5d2eb114c679ad8c462c2b58aed